.github/workflows/release.yml

```diff
@@ -83,7 +83,11 @@ jobs:
 
     - name: Install dependencies (Linux)
       if: startsWith(matrix.platform, 'ubuntu-')
-      run: sudo apt-get -y install ccache cmake make gcc g++ flex libfl-dev bison libpcap-dev libssl-dev python3 python3-dev python3-setuptools swig zlib1g-dev zip libmaxminddb-dev
+      run: |
+        sudo apt-get -y install ccache cmake make gcc g++ flex libfl-dev bison libpcap-dev libssl-dev python3 python3-dev python3-setuptools swig zlib1g-dev zip libmaxminddb-dev
+        sudo find /usr/lib \( -name libpcap.so\* -o -name libmaxminddb.so\* \) -delete
+        echo "extra_config=-D ZLIB_USE_STATIC_LIBS=ON" >> $GITHUB_OUTPUT
```

Comment:
I'm not proud of this hack, but here's what's going on. If I invoke _just_ `-D ZLIB_USE_STATIC_LIBS=ON` in the `configure` step below that's used on both Linux and macOS, it would then complain on macOS that it couldn't find the static lib for ZLIB because it's in a `Cellar` Homebrew directory. I could make _that_ happy by using the `ZLIB_ROOT` hint on macOS to point to the Cellar directory, but we were comfortable with linking ZLIB dynamically on macOS before because it ships with the OS. Therefore I'm defining this extra config that gets added to the `configure` command line below on Linux but resolves to an empty string on macOS.